src/Sylius/Bundle/AdminBundle/Resources/views/product_association_type/form/translations.html.twig

```diff
@@ -0,0 +1,14 @@
+{% import '@SyliusAdmin/Shared/Helper/translations.html.twig' as _translations %}
+
+<div class="bg-white mb-3">
+    {% set body %}
+        {% verbatim %}
+            <div class="row"">
+                <div class="col-12">
+                    {{ form_row(translationForm.name, sylius_test_form_attribute('name', locale)) }}
```

Comment:
We should slightly refactor translations to add hook for every field, it should be done in a separate PR for all resources already implemented

Reply:
Agreed